Assets/HoloToolkit-Examples/MotionControllers-GrabMechanics/Textures/MRTK_Logo.png.meta

```diff
@@ -0,0 +1,94 @@
+fileFormatVersion: 2
+guid: f721996453d888a4db83f0f9f1a4eb7c
```

Comment:
We should move this into it's own folder most likely, seeing as it's for the toolit in general.

Reply:
Sure - what folder should it be in?

Reply:
I've moved it to HoloToolKit/UI/Textures/

Reply:
That's exactly what I was thinking! :)